lib/builder/components/SettingsChart.js

```diff
@@ -158,8 +138,15 @@ class SettingsChart extends Component {
       chart: type
     };
 
+    let timer = null
```

Comment:
What is the reason for creating that function in `render` method. Please, have in mind that for each render the function will be re-defined as well as timer variable.

Please consider:
- Moving the function to class.
- For debouncing use lodash method as is already project dependency.